microsoft-azure-sb/3.1/modules/ROOT/pages/azure-service-bus-connector-studio.adoc

```diff
@@ -127,6 +127,12 @@ Unless the log file path was customized in the app’s log file (`log4j2.xml`),
 `MULE_HOME/logs/<app-name>.log`
 For more information about the app log, see xref:mule-runtime::logging-in-mule.adoc[Configuring Logging].
 
+=== Reduce Excessive Logging
+
+When using the Message Listener source, the Azure Service Bus library emits an INFO-level log message approximately every 30 seconds, indicating that no new messages were received. To avoid this excessive logging, add the following AsyncLogger element to the `log4j.xml` file:
```

Comment:
```suggestion
When using the *Message Listener* source, the Azure Service Bus library emits an INFO-level log message approximately every 30 seconds, indicating that no new messages were received. To avoid this excessive logging, add the following AsyncLogger element to the `log4j.xml` file:
```